apps/admission-webhook-k8s/README.md

```diff
@@ -0,0 +1,150 @@
+# Webhook options
+
+## Prerequisite
+
+This is only possible for local setup:
+- clone project
+- use [`to-local.sh`](../../to-local.sh)
+- then run all the commands in the `deployments-k8s` folder
+
+Usual way to apply example:
+1. set up spire
+2. set up basic example
+3. set up example
+
+In order for the changes below to work, they must be made before step 2.
+
+As the first step for all cases below delete the spiffeid Spire webhook template if it exists:
+```bash
+  $ kubectl delete clusterspiffeid.spire.spiffe.io/nsm-workloads-webhook
```

Comment:
Please, remove `$` symbols and empty spaces at the beginning of commands, so that people can copy and run the whole command.

Reply:
Done